pages/agenda/2018.tsx

```diff
@@ -0,0 +1,280 @@
+import * as React from 'react'
+import { Fragment } from 'react'
+import AgendaPage, { AgendaPageParameters, AgendaPageProps } from '../../components/agendaPage'
+import withPageMetadata from '../../components/global/withPageMetadata'
+import ResponsiveVideo from '../../components/responsiveVideo'
+import Sponsors from '../../components/sponsors'
+import From2018 from '../../config/2018'
+import { SponsorType } from '../../config/types'
+
+class Agenda2018 extends React.Component<AgendaPageProps> {
+  static getAgendaPageParams(): AgendaPageParameters {
+    return {
+      conferenceInstance: '2018',
+      numTracks: 4,
+      sessionsUrl: '/static/agenda/2018.json',
+      sponsors: [],
+    }
+  }
+
+  render() {
+    const SessionCell = this.props.SessionCell
+    return (
+      <Fragment>
+        <div style={{ overflow: 'auto' }}>
+          <table className="agenda-row table">
+            <thead>
+              <tr>
+                <th style={{ width: '4%' }} />
+                <th style={{ width: '24%' }}>
+                  <strong className="room">RR4</strong>
```

Comment:
semantically it would make more sense to not have strong and add the visual styling needed to th

Reply:
Agreed, feel free to correct / add to backlog